tiles.h

```diff
@@ -96,7 +87,6 @@ static const struct tile_s* tiles[TileCount] =
    &mountainTile,
```

Comment:
if this array is the reason you are getting that unused error you can extern it and define in a c file, the same for all those other structs. It does seem kinda bad to do it that way but I don't know if there is a better solution